metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_config.py

```diff
@@ -287,6 +296,17 @@ def validate_column_lineage(cls, v: bool, values: Dict[str, Any]) -> bool:
         description="Option to exclude empty projects from being ingested.",
     )
 
+    owner_lable_character_mapping: Dict[str, str] = Field(
+        default={},
+        description="A mapping of bigquery owner label character to datahub owner character."
+        "Provided mapping will get added to default mapping.",
+    )
+
+    owner_key_pattern: str = Field(
+        default="_owner_email",
+        description="A pattern which defines what identifies an owner label.",
+    )
+
```

Comment:
I'm looking at the BigQuery datahub source docs and seeing `capture_table_labels_as_tags` and think we should model this to be similar. So something like:

```
capture_table_owner_label_as_owner: 
    enabled: true
    # we want them to be able to define a mapping, so our BQ label -> datahub owner ingestion is generic
    label_character_mapping:
        - "_": "."
        - "-": "@"
        - "__": "_"
        - "--": "-"
        - "_-": "#"
        - "-_": " "

    # we want them to be able to define what identifies an owner label, so it is also generic
    owner_key_pattern: "_owner_email"
```

Reply:
https://datahubproject.io/docs/generated/ingestion/sources/bigquery#config-details